arch/arm/src/cxd56xx/cxd56_farapi.c

```diff
@@ -196,14 +196,14 @@ void farapi_main(int id, void *arg, struct modulelist_s *mlist)
     {
       /* Save the current cpuset */
 
-      sched_getaffinity(getpid(), sizeof(cpu_set_t), &cpuset0);
+      nxsched_get_affinity(getpid(), sizeof(cpu_set_t), &cpuset0);
```

Comment:
@pkarashchenko since the POSIX API name is sched_getaffinity/setaffinity, shouldn't be better to create nxsched_getaffinity/setaffinity?

Reply:
I do not create a new API and just move things around in this PR

Reply:
POSIX sched_getaffinity/setaffinity is kept unchanged, but just moved to libc

Reply:
This is a kernel code. No need to call user space APIs here